pepper-apis/src/test/java/org/broadinstitute/ddp/route/CreateTemporaryUserRouteTest.java

```diff
@@ -34,13 +36,15 @@
     private static Set<String> usersToDelete;
     private static Gson gson;
     private static String url;
+    private static String auth0Domain;
 
     @BeforeClass
     public static void setup() {
         TransactionWrapper.useTxn(handle -> testData = TestDataSetupUtil.generateBasicUserTestData(handle));
         url = RouteTestUtil.getTestingBaseUrl() + RouteConstants.API.TEMP_USERS;
         usersToDelete = new HashSet<>();
         gson = new Gson();
+        auth0Domain = ConfigManager.getInstance().getConfig().getConfig(ConfigFile.AUTH0).getString(ConfigFile.DOMAIN);
```

Comment:
Integration tests have access to the config object already, you can use `RoutTestUtils.getConfig()`.

Reply:
Done.